### DISCUSSION TITLE
download nova launcher beta

### DISCUSSION THREAD

**Discussion by Karthikn341** [ImranR98/Obtainium#1088] [Q&A] [answered]
https://novalauncher.com/beta

`https://teslacoilapps.com/tesladirect/download.pl?packageName=com.teslacoilsw.launcher&amp;betaType=public`

If I open this on browser, nova launcher beta latest version get downloaded.. how to add this on obtainium?

**Comment by ImranR98** [comment, marked as answer]
See #1048
